Document CPU limit mode in AMS (#149)
anbox-cloud/discourse-docs: ref/ams-configuration.md

| `registry.mode` | string | pull | Mode the registry client in AMS operates in. Possible values are: manual, pull, push|
| `registry.url` | string | - | URL of the application registry to use |
| `scheduler.strategy` | string | spread | Strategy the internal container scheduler in AMS is using to distribute container across available LXD nodes. Possible values are: `binpack`, `spread` |

> | `cpu.limit_mode` | string | scheduler | Mode AMS uses to limit CPU access for a container. Have a look [here]](https://discourse.ubuntu.com/t/about-performance/29416) for more details. Possible values are: `scheduler`, `pinning` |

Reviewer: Please move this up (the entries are alphabetical).

Reviewer: Still at the bottom.

Author: Fixed now

> | `cpu.limit_mode` | string | scheduler | Mode AMS uses to limit CPU access for a container. Have a look [here]](https://discourse.ubuntu.com/t/about-performance/29416) for more details. Possible values are: `scheduler`, `pinning` |

Reviewer: ```suggestion
| `cpu.limit_mode` | string | scheduler | The mode AMS uses to limit CPU access for a container. See [About performance](https://discourse.ubuntu.com/t/about-performance/29416) for details. Possible values are: `scheduler`, `pinning` |
```
